### PR TITLE
[15.0][FIX] stock_picking_info_lot: no lot required check if no done quantities

### PR DESCRIPTION
**Pull request by dreispt**
- [FIX] stock_picking_info_lot: no lot required check if no done quantities

### CODE REVIEW

File: stock_picking_info_lot/views/stock_move_line_views.xml [hunk 1]
```diff
@@ -65,7 +64,6 @@
                     name="lot_info"
                     attrs="{
                        'readonly': [('lot_info_usage', '=', 'no')],
-                       'required': [('lot_info_usage', '=', 'required')],
```
Comment (rousseldenis) [inline stock_picking_info_lot/views/stock_move_line_views.xml:68]: @dreispt 
Maybe you can set a field that compute the 'required' condition of lot_info_usage for form, so user can see if the field is required when he filled in a quantity ?

**Comment by dreispt** [comment]
@rousseldenis Is any of the comments blocking for a merge? Thanks

**Review by rousseldenis — COMMENTED** [review]
@dreispt Could you just add tests ?

**Comment by rousseldenis** [comment]
@dreispt

**Pull request closed without merging**